docker-compose.yml

```diff
@@ -32,4 +32,4 @@ services:
   lido_terra:
     build: .
     env_file:
-      ./docker/env/.lido_terra.env
+      ./docker/env/.lido_terra.testnet.env
```

Comment:
i think we should leave name .lido_terra.env in the config. And the operator will put out the file himself by copying or symlinking

Reply:
will do